tfjs-layers/src/layers/preprocessing/random_width_test.ts

```diff
@@ -0,0 +1,161 @@
+/**
+ * @license
+ * Copyright 2023 CodeSmith LLC
+ *
+ * Use of this source code is governed by an MIT-style
+ * license that can be found in the LICENSE file or at
+ * https://opensource.org/licenses/MIT.
+ * =============================================================================
+ */
+
+/**
+ * Unit Tests for random width layer.
+ */
+
+import { Tensor, reshape, range, image, Rank, zeros, randomUniform, tensor } from '@tensorflow/tfjs-core';
+import { describeMathCPUAndGPU, expectTensorsClose } from '../../utils/test_utils';
+
+import { RandomWidth, RandomWidthArgs } from './random_width';
+
+describeMathCPUAndGPU('RandomWidth Layer', () => {
+  it('Returns correct, randomly scaled width of Rank 3 Tensor', () => {
+    const rangeTensor = range(0, 16);
+    const inputTensor = reshape(rangeTensor, [4,4,1]);
```

Comment:
Minor nit since `rangeTensor` only seems to be used once.
```suggestion
    const inputTensor = range(0, 16).reshape([4, 4, 1]);
```